src/pages/auth/components/Login/login.test.tsx

```diff
@@ -0,0 +1,91 @@
+import Login from './login';
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom'
+import * as React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import { GoogleOAuthProvider } from '@react-oauth/google';
+import { AuthProvider } from '../../../../contexts/Auth';
+
+describe('Login page tests', () => {
+
+    test('login page - should show two sign in options', () => {
+      render( 
+        <BrowserRouter>
+            <GoogleOAuthProvider clientId='test-client-id'>
+                <Login /> 
+            </GoogleOAuthProvider>
+        </BrowserRouter>
+        );
+      expect(screen.getByText(/Welcome Back/)).toBeInTheDocument();
+      expect(screen.getByText(/Sign in with Google/)).toBeInTheDocument();
+      expect(screen.getByText(/Sign in with Email/)).toBeInTheDocument();
+    });
+
+    test('login page - clicking google sign in option opens new tab', () => {
+      //needs to be revisited
+      const openTab = jest.spyOn(global, "open");
+      const wrapper = render( 
```

Comment:
Why's wrapper not being used?

Reply:
Needs to revisited cause it actually doesn't test if the window is being opened